package.json

```diff
@@ -33,7 +33,7 @@
   "license": "MIT",
   "dependencies": {
     "@truffle/config": "1.2.33",
-    "@truffle/hdwallet-provider": "1.1.1",
+    "@truffle/hdwallet-provider": "^1.1.1",
```

Comment:
Please, remove ^ versions, for easy management of updates, we must provide fixed versions